pyomo/opt/plugins/sol.py

```diff
@@ -200,8 +200,8 @@ def _load(self, fin, res, soln, suffixes):
                 if (unmasked_kind & 4) == 4:
                     convert_function = float
                 nvalues = int(line[2])
-                namelen = int(line[3])
-                tablen = int(line[4])
+                # namelen = int(line[3])
+                # tablen = int(line[4])
```

Comment:
@blnicho: @mrmundt removed them, and I argued to put them back because the SOL reader is not well documented and this at least provided some hints as to what the expected fields are in the SOL foles.

Reply:
@jsiirola yeah that's a good point